acquia_cms.profile

```diff
@@ -287,3 +288,15 @@ function install_acms_set_favicon() {
     ])
     ->save(TRUE);
 }
+
+/**
+ * Update config ignore settings.
+ */
+function acquia_cms_update_8001() {
+  $module_path = \Drupal::moduleHandler()->getModule('acquia_cms')->getPath();
+  $config_optional = $module_path . '/config/optional';
+  $source_optional_dir = new FileStorage($config_optional);
+
+  $config_storage = \Drupal::service('config.storage');
+  $config_storage->write('config_ignore.settings', $source_optional_dir->read('config_ignore.settings'));
```

Comment:
@chandan-singh7929 I think we can’t re-import the complete config_ignore config, as others might have added/removed some configs. Those changes will be then vanished. We've add the config changes in config_ignore config (if those are not already present).